<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeomWindow</class>
 <widget class="QMainWindow" name="GeomWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>750</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift SemiBold</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Geometry of Chamber</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="tools_layout">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="h1_layout">
         <item>
          <widget class="QLabel" name="h1_label">
           <property name="text">
            <string>1st collim.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="h1_box">
           <property name="suffix">
            <string> mm</string>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="h2_layout">
         <item>
          <widget class="QLabel" name="h2_label">
           <property name="text">
            <string>2nd collim.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="h2_box">
           <property name="suffix">
            <string> mm</string>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="d1_layout">
         <item>
          <widget class="QLabel" name="d1_label">
           <property name="text">
            <string>D1:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="d1_box">
           <property name="suffix">
            <string> mm</string>
           </property>
           <property name="maximum">
            <double>99999.990000000005239</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="d2_layout">
         <item>
          <widget class="QLabel" name="d2_label">
           <property name="text">
            <string>D2:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="d2_box">
           <property name="suffix">
            <string> mm</string>
           </property>
           <property name="maximum">
            <double>99999.990000000005239</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="telescope1_layout">
         <item>
          <widget class="QLabel" name="telescope1_label">
           <property name="text">
            <string>1st telescope:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="telescope1_slider">
           <property name="maximum">
            <number>180</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="telescope2_layout">
         <item>
          <widget class="QLabel" name="telescope2_label">
           <property name="text">
            <string>2nd telescope:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="telescope2_slider">
           <property name="maximum">
            <number>180</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="telescope3_layout">
         <item>
          <widget class="QLabel" name="telescope3_label">
           <property name="text">
            <string>3rd telescope:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="telescope3_slider">
           <property name="maximum">
            <number>180</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="target_layout">
         <item>
          <widget class="QLabel" name="target_label">
           <property name="text">
            <string>Target angle:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="target_slider">
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="singleStep">
            <number>45</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="remove_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>Remove labels</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="settings_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="output"/>
       </item>
       <item>
        <widget class="QPushButton" name="collimator_optics_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>Collimator optics</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="telescope_optics_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>Telescope optics</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="author_label">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Chamber, LLENR app by TheKoke</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="matplotlib_layout">
      <property name="minimumSize">
       <size>
        <width>780</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
